backend/constants.js

```diff
@@ -4,3 +4,4 @@ export const JUDGE = 'JUDGE'
 export const IMAGE_ENTRY = 0x1
 export const VIDEO_ENTRY = 0x2
 export const OTHER_ENTRY = 0x3
+export const ALLOWED_VIDEO_PROVIDERS = new Set(['youtube', 'vimeo'])
```

Comment:
Is there an immutable set?

Reply:
apparently you can't do this w/o a dependency so it's staying as is.